src/entry.py

```diff
@@ -273,7 +273,9 @@ def process_files(
 
         score = 0
         if evaluation_config is not None:
-            score = evaluate_concatenated_response(omr_response, evaluation_config)
+            score = evaluate_concatenated_response(
+                omr_response, evaluation_config, file_path, outputs_namespace.paths.evaluation_dir
+            )
```

Comment:
this is good for now, maybe in future we will call this function from the evaluation_config instance i.e. `evaluation_config_instance.evaluate_concatenated_response(omr_response, file_path)` or something similar where the evaluation_dir is part of the instance